File: year2/mpp/java/MotorcycleContest/src/main/java/domain/entities/Participant.java
package domain.entities;

public class Participant extends Entity<Integer> {
    private String name;
    private Integer raceId;
    private Integer teamId;

    public Participant() {
        this.name = null;
        this.raceId = null;
        this.teamId = null;
    }

    public Participant(String name, Integer raceId, Integer teamId) {
        this.name = name;
        this.raceId = raceId;
        this.teamId = teamId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getRaceId() {
        return raceId;
    }

    public void setRaceId(Integer raceId) {
        this.raceId = raceId;
    }

    public Integer getTeamId() {
        return teamId;
    }

    public void setTeamId(Integer teamId) {
        this.teamId = teamId;
    }

    @Override
    public String toString() {
        return "Participant " + getId() + ", name " + name + ", race " + raceId + ", team " + teamId + "; ";
    }
}

File: year2/mpp/c#/MotorcycleContest/MotorcycleContest/Domain/Entities/Participant.cs
using System;

namespace MotorcycleContest.Domain.Entities
{
    public class Participant : Entity<int>
    {
        public String Name { get; set; }
        public int RaceId { get; set; }
        public int TeamId { get; set; }

        public Participant()
        {
            this.Name = null;
            this.RaceId = -1;
            this.TeamId = -1;
        }

        public Participant(String name, int raceId, int teamId)
        {
            this.Name = name;
            this.RaceId = raceId;
            this.TeamId = teamId;
        }

        public override string ToString()
        {
            return "Participant " + ID + ", name " + Name + ", race " + RaceId + ", team " + TeamId + "; ";
        }
    }
}
